app/src/main/java/io/homeassistant/companion/android/notifications/MessagingManager.kt

```diff
@@ -1935,6 +1943,29 @@ class MessagingManager @Inject constructor(
         }
     }
 
+    private fun setAppLock(data: Map<String, String>) {
+        try {
+            if (data.containsKey("app_lock_enabled")) {
+                runBlocking {
+                    authenticationUseCase.setLockEnabled(data["app_lock_enabled"]!!.toBooleanStrict())
```

Comment:
Should we add a default if the value we get is `null` ?

Reply:
The intention was for all app-lock related settings to be adjustable in a single command, to avoid having to send multiple commands to the same device. Parameters not present in the notification should be ignored and their settings kept as is.

Though I guess technically the key might be present and the value still be `null`. I'll add some more explicit checks here to ensure the values that are set are sensible.

Reply:
> Though I guess technically the key might be present and the value still be `null`.

it wouldn't be the first time it happened 😄